packages/block-editor/src/components/use-block-sync/block-manager.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import { v4 as uuid } from 'uuid';
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useRef } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import {
+	setupEntityInstance,
+	isDuplicateEntityInstance,
+	removeEntityInstance,
+} from './entity-instances';
+
+class BlockManager {
```

Comment:
I really like how we've consolidated all of the fields, constants, and methods related to entity and local block id management. It feels natural with this class-based approach.

Reply:
@youknowriad could you elaborate on what `create` functions are?